linalg/blockoperator.cpp

```diff
@@ -49,9 +49,9 @@ BlockOperator::BlockOperator(const Array<int> & row_offsets_,
    col_offsets.MakeRef(col_offsets_);
 }
 
-void BlockOperator::SetDiagonalBlock(int iblock, Operator *op, double c)
+void BlockOperator::SetDiagonalBlock(int iblock, Operator *opt, double c)
```

Comment:
Is there a a reason why we do not stick to the convention of putting a an underscore as a suffix for input arguments? This is e.g. utilized in several constructors.

Reply:
Unfortunately, we have not been able to agree on a convention so this is left up to the individual developer.

Reply:
Out of curiosity, is there something wrong with the underscore suffix?

Reply:
There's nothing "wrong" with trailing underscores. It's just a matter of taste. Some of our miniapps, in electrodynamics for example, use trailing underscores to distinguish class member data. Some of our developers prefer to avoid trailing underscores in favor of more descriptive variable names.

Adopting any convention at this stage would involve many changes to various classes so people are understandably reluctant.

Reply:
Understandable. Thanks for the elaboration!